WcaOnRails/app/views/layouts/_navigation.html.erb

```diff
@@ -3,6 +3,25 @@
     <div class="navbar-brand">
       <a href="<%= root_path %>"><%= image_tag "wca_logo.svg" %></a>
       <a href="<%= root_path %>"><span>World Cube Association</span></a>
+      |
```

Comment:
Bootstrap has some [responsive utilities](https://getbootstrap.com/docs/3.3/css/#responsive-utilities), maybe make this a `<span class="hidden-xs hidden-sm">|</span>`, and add a `<span class="visible-xs-block visible-sm-block"></span>` (to add a `display:block` item in the middle of the flow and trigger a "line break" only for small screens)